src/i18n/strings/de_DE.json

```diff
@@ -17,6 +17,9 @@
     "Failed to start": "Start fehlgeschlagen",
     "%(brand)s Desktop (%(platformName)s)": "%(brand)s Desktop (%(platformName)s)",
     "%(appName)s (%(browserName)s, %(osName)s)": "%(appName)s (%(browserName)s, %(osName)s)",
+    "Failed to load Jitsi widget": "Das Jitsi Widget konnte nicht geladen werden",
+    "Jitsi Video Conference": "Videokonferenz",
+    "Join Conference": "Videokonferenz beitreten",
     "Please install <chromeLink>Chrome</chromeLink>, <firefoxLink>Firefox</firefoxLink>, or <safariLink>Safari</safariLink> for the best experience.": "Bitte installiere <chromeLink>Chrome</chromeLink>, <firefoxLink>Firefox</firefoxLink> oder <safariLink>Safari</safariLink> für das beste Erlebnis.",
```

Comment:
As far as I understand, these changes are actually not allowed manually. I would like to keep them during the review phase as it is otherwise hard to test. I can remove them before a merge and submit them via the translation portal.

Reply:
Yes, that is why https://github.com/vector-im/element-web/actions/runs/3183049256/jobs/5209163177 fails.

You should just run `yarn run i18n` after modifying translatable strings. For tests you may modify them locally.